docs/DeveloperGuide.md

```diff
@@ -2,7 +2,14 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Class diagram for v2.0
+![Class Diagram](https://github.com/AY2021S2-CS2113-T10-2/tp/blob/master/docs/assets/Class Diagram2.svg)
+
+### Sequence digram for v2.0
+### Sequence digram for listAllLocations
+![Sequence Digram of listAllLocations](https://github.com/AY2021S2-CS2113-T10-2/tp/blob/master/docs/assets/Sequence%20Digram.JPG)
+### Sequence digram for findNearest
+![Sequence Digram of findNearest](https://github.com/AY2021S2-CS2113-T10-2/tp/blob/master/docs/assets/findNearest.png)
```

Comment:
I think an accompanying detailed description of the sequence diagram would be appropriate here - I'm not sure if the description is included in the .jpg file, though, but some explanation of the contents of the diagram would be nice!
- another minor issue: diagram is spelt wrongly
![image](https://user-images.githubusercontent.com/57757704/113081419-38319080-920b-11eb-9803-f84e16cfa482.png)